cypress/page/payment-step.page.ts

```diff
@@ -5,20 +5,20 @@ class PaymentStepPage {
 
   constructor() {
     this.payBank = ".bankwire";
-    this.order = ".cart_navigation.clearfix > button";
-    this.messageOrder = "#center_column > div > p > strong";
+    this.order = ".cart_navigation";
+    this.messageOrder = ".dark";
   }
 
   public goToPaymentStep(): void {
     cy.get(this.payBank).click();
   }
 
   public confirmOrder(): void {
-    cy.get(this.order).click();
+    cy.get(this.order).contains("I confirm my order").click();
   }
 
   public getConfirmMessageOrder() {
-    return cy.get(this.messageOrder);
+    return cy.get("strong").get(this.messageOrder).contains("Your order on My Store is complete.");
```

Comment:
this change is not the best approach, since you are attaching the get to one specific message, if the message change this method will not return the confimMessage, it is better the previous selector than this approach